Algorithms/Searching/fibonacci_search.cpp

```diff
@@ -0,0 +1,45 @@
+#include<iostream>
+using namespace std;
+void FibonacciSearch(int *a, int start, int end, int *fib, int index, int item) {

```

Comment:
do we need to use pointers?

Reply:
As we are using recursion, so using pointer will reduce the space complexity.